app/frontends/stac-item.ts

```diff
@@ -126,11 +129,19 @@ export class HarmonyItem {
    * @param end - Data end datetime
    *
    */
-  addTemporalExtent(start: string, end: string): void {
-    // Validate
-    this.setProperty('start_datetime', start);
-    this.setProperty('end_datetime', end);
-    this.setProperty('datetime', start);
+  addTemporalExtent(start: Date | string, end: Date | string): void {
+    let startString = start as string;
```

Comment:
This logic is a little confusing to me. I think it might be more straight-forward like this
```
const startString = typeof start === 'string' ? start : start.toISOString();

Reply:
Done